### DISCUSSION TITLE
Does mcp use forge or fabric?

### DISCUSSION THREAD

**Discussion by vtf6259** [Hexeption/MCP-Reborn#261] [General]
Does mcp use forge or fabric because i want to use it on my mc account?
i guess it uses forge idk.

**Comment by APie357** [comment]
MCP is modifying the official jar, not creating mods for a mod loader. It decompiles Minecraft and lets you modify it. Since this contains Minecraft's code, you **may not distribute it**. If you would like to use your Minecraft account with this mod, please see the instructions on exporting the mod in `README.md`.
